src/build_workflow/build_args.py

```diff
@@ -10,10 +10,22 @@
 
 
 class BuildArgs:
+    SUPPORTED_PLATFORMS = [
+        "linux"
+    ]
+    SUPPORTED_ARCHITECTURES = [
+        "x64",
+        "x86_64",
```

Comment:
Are x64 and x86_64 a dup here? I never know which one we pass from where, I think the two here should be `x64` and `arm64`, since this is what we return from `current_architecture`.